Lib/test/test_future.py

```diff
@@ -310,19 +318,13 @@ def test_annotations(self):
         eq("f'{x}'")
         eq("f'{x!r}'")
         eq("f'{x!a}'")
-        eq('(yield from outside_of_generator)')
```

Comment:
Can we move all these expressions to `test_annotations_forbidden` so is clear that we are not missing any case?